PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl/DPad.rbxmx

```diff
@@ -12,6 +12,7 @@
 --]]
 
 local Players = game:GetService('Players')
+local GUIS = game:GetService('GuiService')
```

Comment:
Could we name this GuiService? The other services are referenced in a way such that the name is basically the same